sway/input/seatop_default.c

```diff
@@ -342,8 +362,7 @@ static void handle_button(struct sway_seat *seat, uint32_t time_msec,
 	}
 
 	// Handle tiling resize via mod
-	bool mod_pressed = keyboard &&
```

Comment:
Minor style changed batched here; `modifiers` is guaranteed to be `0` if `keyboard` is null.